includes/CMB2_Base.php

```diff
@@ -379,6 +379,43 @@ public static function maybe_hook( $val, $hook_name, $hook_function ) {
 	}
 
 	/**
+	 * Arguments for custom wp_kses sanitization/validation calls.
+	 *
+	 * @since  2.2.4
+	 * @param  string $type The field type being escaped.
+	 * @return array        An array of wp_kses HTML parameters allowed.
+	 */
+	public function kses_args( $type ) {
+		$params = array();
+
+		switch ( $type ) {
+			case 'group_wrap_attributes':
+				$params = array(
+					'class' => array(),
+					'style' => array(),
+				);
+				break;
+			case 'remove_disabled':
+				$params = array(
+					'disabled' => array(),
+				);
+		}
+
+		return $params;
+	}
+
+	/**
+	 * Helper function that returns whether a field group is repeatable or not.
+	 *
+	 * @since  2.2.4
+	 * @param  object $field_group The field group.
+	 * @return boolean
+	 */
+	public function is_repeatable( $field_group ) {
```

Comment:
> It's a couple less characters to type.

but... it's not. :)
![image](https://cloud.githubusercontent.com/assets/1098900/24715452/5ab6ec92-19f9-11e7-8c4e-ec5711186533.png)